src/og/layer/CanvasTiles.js

```diff
@@ -247,7 +248,9 @@ class CanvasTiles extends Layer {
     applyMaterial(material) {
         if (material.isReady) {
             return [0, 0, 1, 1];
-        } else {
+        } else if (material.segment.tileZoom <= this.minNativeZoom) {
```

Comment:
I think it must be `material.segment.tileZoom < this.minNativeZoom`  minNativeZoom is included.